cli/crates/cli/tests/auth.rs

```diff
@@ -0,0 +1,95 @@
+mod utils;
+
+use serde_json::{json, Value};
+use utils::environment::Environment;
+
+const ISSUER_URL: &str = "https://some.issuer.test";
+const JWT_SECRET: &str = "topsecret";
+
+#[test]
+fn jwt_provider() {
+    const GROUP: &str = "backend";
+    const SCHEMA: &str = const_format::formatcp!(
```

Comment:
That's in fact how I started, but it would break interpolation and make the test harder to understand IMO.

Reply:
Not sure I understand why we're formatting here at all, aren't all of these static for the entire test?

Reply:
I moved the schema to a file and used env variables to avoid duplication of the issuer URL and JWT secret.

I still think that the test is less clear now and that using files didn't add anything other than maybe proper code formatting in the IDE. But on the other hand, I don't have strong feelings about it either, so let's keep it consistent.